ale/base/label_isis.py

```diff
@@ -134,3 +134,15 @@ def  exposure_duration(self):
             # if no units are available, assume the exposure duration is given in milliseconds
             exposure_duration = self.label['IsisCube']['Instrument']['ExposureDuration'].value * 0.001
         return exposure_duration
+
+    @property
+    """
+    The line exposure duration of the image, in seconds
+
+    Returns
+    -------
+    : float
+      Line exposure duration in seconds
+    """
```

Comment:
This needs to go under the `def line_exposure_duration` line

Reply:
yep, just realized that